src/preprocessing/passes/ackermann.h

```diff
@@ -64,6 +68,14 @@ class Ackermann : public PreprocessingPass
   /* Map each function term to the new Skolem variable created by
    * ackermannization */
   theory::SubstitutionMap d_funcToSkolem;
+  /* Map each uninterpreted sort to the new Skolem variable created by
+   * ackermannization */
+  theory::SubstitutionMap d_sortsToSkolem;
```

Comment:
Do we map each sort to a variable? Or each variable of a sort to a BV variable?

Reply:
Oh, we map each variable of a sort to a BV variable. Maybe I need to reconsider for a better name.